webapp/components/wpt-results.html

```diff
@@ -326,6 +368,17 @@
         this.refreshDisplayedNodes();
       }
 
+      async fetchManifest(path) {
+        if (!this.computePathIsATestFile(path)) {
+          return;
+        }
+        this.manifest = null;
```

Comment:
Given the early return I wonder if this is necessary, since `this.manifest` won't always be set anyway. Initialize it in a constructor or something if it matters?

Reply:
It's for (instantly) clearing out previous values (while the new value loads).